stl/inc/xlocmon

```diff
@@ -689,8 +689,8 @@ protected:
         const ctype<_Elem>& _Ctype_fac = _STD use_facet<ctype<_Elem>>(_Iosbase.getloc());
         const _Elem _E0                = _Ctype_fac.widen('0');
 
-        string_type _Val2(_Count, _Elem{});
-        _Ctype_fac.widen(&_Buf[0], &_Buf[_Count], &_Val2[0]);
+        string_type _Val2(static_cast<size_t>(_Count), _Elem{});

```

Comment:
I don't know if `_Elem` here is an arbitrary char-like type or if it's "a member of the set of types
containing `char`, `wchar_t`, and any other implementation-defined character types that meet the requirements
for a character on which any of the iostream components can be instantiated." (which we control). If the former, we should really `_Elem()` instead of `_Elem{}` here.